prover/protocol/distributed/xcomp/xcomp.go

```diff
@@ -0,0 +1,144 @@
+package xcomp
+
+import (
+	"fmt"
+
+	"github.com/consensys/gnark/frontend"
+	"github.com/consensys/linea-monorepo/prover/maths/field"
+	"github.com/consensys/linea-monorepo/prover/protocol/accessors"
+	"github.com/consensys/linea-monorepo/prover/protocol/wizard"
+	"github.com/consensys/linea-monorepo/prover/utils/collection"
+)
+
+// GetCrossComp generates an (empty) compiledIOP object that is handling the crosse checks
+// for example, the global sum over logDerivativeSum is zero.
+func GetCrossComp(vRuntimes []*wizard.VerifierRuntime) *wizard.CompiledIOP {
```

Comment:
Yes, to the idea to construct it as a verifierAction as it is simple and small enough. But the API of the function cannot make sense as is. We will fix this when integrating with the conglomeration that I wrote.